src/main/java/io/vertx/core/http/HttpServerRequest.java

```diff
@@ -170,6 +169,18 @@ default String getParam(String paramName) {
     return params().get(paramName);
   }
 
+  /**
+   * Return the first param value or defaultValue if the query param is not provided.
+   *
+   * @param paramName    the param name
+   * @param defaultValue the default value, must be non-null
+   * @return the param value or default value
```

Comment:
"the param value or `@code{defaultValue}` when not present"

Reply:
Done.